readme.md

```diff
@@ -76,6 +76,7 @@ Please raise a [Pull-Request](https://github.com/notthetup/awesome-webaudio/pull
 - [mod-synth.io](https://github.com/LowwwLtd/mod-synth.io) - Create your own modular synthesizer, or emulate different synths.
 - [GridSound](https://gridsound.github.io) - A work-in-progress DAW (Digital Audio Workstation).
 - [Learning Music](https://learningmusic.ableton.com/) - Learn the basics of music making.
+- [Super Oscillator](https://github.com/lukehorvat/super-oscillator) - An interactive, 3D music synthesizer for the Web!
```

Comment:
Could you change the end of the sentence from a `!` to `.`? It's a awesome-list regulation thingy.. :(

Reply:
Heh. 😆  Done.